### PR TITLE
Quality of life changes

### PR DESCRIPTION
**Pull request by whopavan**
Added a few quality of life changes.

- Added launch.json file
- Separated theme data to different file (easier to maintain when it starts getting bigger)
- Updated settings page to a much cleaner UI
- Added theme switcher to home screen

Let me know if any changes to this are required. Added few screenshots below

image
image

### CODE REVIEW

File: lib/app/settings/presentation/settings_view.dart [hunk 1]
```diff
@@ -30,25 +30,29 @@ class SettingsView extends ConsumerWidget {
...
-                    ),
-                    const SizedBox(height: 8),
                     SettingDurationCard(
                       duration: state.focusDuration,
                       icon: '🧘‍♂️',
                       title: 'Focus Duration',
                       onDurationChanged: notifier.updateFocusDuration,
                     ),
                     const SizedBox(height: 8),
+                    const _SettingsDivider(),
```
Comment (rkishan516) [inline lib/app/settings/presentation/settings_view.dart]: @whopavan I think focus duration and focus autostart toggle should be together. And break duration and break auto start toggle should be together.

Comment (whopavan) [reply lib/app/settings/presentation/settings_view.dart]: Ok. Does this look fine?
image

Comment (rkishan516) [reply lib/app/settings/presentation/settings_view.dart]: Yup

**Comment by whopavan** [comment]
Pushed the change

**Pull request merged**